File: script/new/curriculumn.js
db.curriculumn.remove();

db.curriculumn.insert({
duration : 1,
duration_type : "Week",
from_date : ISODate("2014-05-01T00:00:00Z"),
material : [db.material.find()[0]._id,db.material.find()[1]._id,db.material.find()[2]._id,db.material.find()[3]._id],
mentor : db.mentor.find()[0]._id,
name : "初心者のためのクラウドサービス",
published_date : ISODate("2014-05-16T18:34:20.588Z"),
to_date : ISODate("2014-05-16T00:00:00Z"),
units : [ ],
action : [db.action.find()[0]._id,db.action.find()[1]._id,db.action.find()[2]._id]
});

db.curriculumn.insert({
duration : 1,
duration_type : "Week",
from_date : ISODate("2014-05-01T00:00:00Z"),
material : [db.material.find()[0]._id,db.material.find()[1]._id],
mentor : db.mentor.find()[0]._id,
name : "How to become Oracle Master",
published_date : ISODate("2014-05-16T18:34:20.588Z"),
to_date : ISODate("2014-05-16T00:00:00Z"),
units : [ ],
action : [db.action.find()[0]._id,db.action.find()[1]._id]
});
db.curriculumn.insert({
duration : 1,
duration_type : "Week",
from_date : ISODate("2014-05-01T00:00:00Z"),
material : [db.material.find()[3]._id,db.material.find()[4]._id],
mentor : db.mentor.find()[0]._id,
name : "Curriculumn name",
published_date : ISODate("2014-05-16T18:34:20.588Z"),
to_date : ISODate("2014-05-16T00:00:00Z"),
units : [ ],
action : [db.action.find()[0]._id,db.action.find()[1]._id]
});
db.curriculumn.insert({
duration : 1,
duration_type : "Week",
from_date : ISODate("2014-05-01T00:00:00Z"),
material : [db.material.find()[5]._id],
mentor : db.mentor.find()[0]._id,
name : "初心者のためのクラウドサービス",
published_date : ISODate("2014-05-16T18:34:20.588Z"),
to_date : ISODate("2014-05-16T00:00:00Z"),
units : [ ],
action : [db.action.find()[0]._id,db.action.find()[1]._id,db.action.find()[2]._id]
});

File: myapp/models/Curriculumn.py
from datetime import datetime

from mongoengine.django.auth import User
from mongoengine.document import Document
from mongoengine.fields import StringField, DateTimeField, ReferenceField, IntField, \
	ListField

from myapp.models.Action import Action
from myapp.models.Category import Category
from myapp.models.Comment import Comment
from myapp.models.Material import Material
from myapp.models.Mentor import Mentor
from myapp.models.Statistic import Statistic
from myapp.models.Unit import Unit


class Curriculumn(Document):
	published_date = DateTimeField(default=datetime.now)
	name = StringField()
	duration = IntField()
	duration_type = StringField()
	category= ReferenceField(Category)
	from_date = DateTimeField(default=datetime.now)
	to_date = DateTimeField(default=datetime.now)
	description = StringField()
	mentor = ReferenceField(Mentor)
	units = ListField(ReferenceField(Unit))
	comments = ListField(ReferenceField(Comment))
	material = ListField(ReferenceField(Material))
	action = ListField(ReferenceField(Action))
	statistic = ReferenceField(Statistic)
	joined_user = ListField(ReferenceField(User))
	meta = {
				'ordering': ['-mentor'],
				'ordering': ['-comments'],
				'ordering': ['-units']
			}
